python/res/enkf/ensemble_config.py

```diff
@@ -41,15 +43,142 @@ class EnsembleConfig(BaseCClass):
     _add_node = ResPrototype("void ensemble_config_add_node( ens_config , enkf_config_node )")
     _summary_key_matcher = ResPrototype("summary_key_matcher_ref ensemble_config_get_summary_key_matcher(ens_config)")
     _add_defined_custom_kw = ResPrototype("enkf_config_node_ref ensemble_config_add_defined_custom_kw(ens_config, char*, integer_hash)")
+    _get_trans_table = ResPrototype("void* ensemble_config_get_trans_table(ens_config)")
+    _add_summary_full = ResPrototype("void ensemble_config_init_SUMMARY_full(ens_config, char*, ecl_sum)")
 
 
+    def get_abs_path(self, file):
```

Comment:
these three helper functions for paths and file name could be moved out of the class and prefixed with _ to indicate that they do not belong to the API.

Reply:
ok